.github/workflows/ci.yaml

```diff
@@ -59,6 +59,10 @@ jobs:
       - name: Install dependencies
         run: composer install --no-interaction --no-progress --no-suggest --prefer-dist
 
+      - name: Static analysis
```

Comment:
This should be moved into the lint workflow as it's own job.

Reply:
Done